health/api/v1/cloud_status.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016: Mirantis Inc.
```

Comment:
This file has wrong name. 

Why it's called cloud_status? if we are descrbing health api? 

By the way this service is going to have relatively small amount of API calls: 

1) get regions 
2) get aggregated score for some period for regions or for services of specific region
3) get aggregated data for regions (for some period of time)
4) get aggregated data for specific region (for some period of time) 

so it's 3-5 simple methods that won't change in future.